### PR TITLE
[PWI] Home

### PR DESCRIPTION
**Pull request by estrattonbailey**
Supersedes #1996

Screenshot 2023-11-27 at 3 41 14 PM
Screenshot 2023-11-27 at 3 41 16 PM
Screenshot 2023-11-27 at 3 41 19 PM
Screenshot 2023-11-27 at 3 41 25 PM

### CODE REVIEW

File: src/view/com/feeds/FeedPage.tsx [hunk 1]
```diff
@@ -195,13 +195,30 @@ export function FeedPage({
...
+    const tabBarPad = 20 + 3 // nav bar padding + border
+    const tabBarText = 16 * fontScale
+    const magic = 7 * fontScale
+    return navBarPad + navBarText + tabBarPad + tabBarText + magic
+  } else {
+    const navBarPad = 16
+    const navBarText = 21 * fontScale
+    const magic = 4 * fontScale
+    return navBarPad + navBarText + magic
   }
```
Comment (estrattonbailey) [inline src/view/com/feeds/FeedPage.tsx:223]: Not proud of this, but it's the best I can come up with right now

File: src/view/com/pager/FeedsTabBar.web.tsx [hunk 2]
```diff
@@ -9,28 +9,82 @@ import {FeedsTabBar as FeedsTabBarMobile} from './FeedsTabBarMobile'
...
+                <View
+                  style={{
+                    top: -8,
+                    backgroundColor: colors.blue3,
+                    width: 8,
+                    height: 8,
+                    borderRadius: 4,
+                  }}
+                />
+              )*/}
```
Comment (estrattonbailey) [inline src/view/com/pager/FeedsTabBar.web.tsx:69]: No good way of hoisting this state out of `FeedPage`

File: src/view/com/pager/FeedsTabBarMobile.tsx [hunk 3]
```diff
@@ -104,8 +109,7 @@ const styles = StyleSheet.create({
     justifyContent: 'space-between',
     alignItems: 'center',
     paddingHorizontal: 18,
-    paddingTop: 8,
-    paddingBottom: 2,
+    paddingVertical: 8,
```
Comment (estrattonbailey) [inline src/view/com/pager/FeedsTabBarMobile.tsx:112]: Needs to look OK without a tab bar

File: src/view/com/feeds/FeedPage.tsx (hunk 1 above)
Comment (pfrazee) [reply src/view/com/feeds/FeedPage.tsx:223]: well, it's clear at least

**Review by pfrazee — APPROVED** [review]
Yeah looks great

**Pull request merged**